### PR TITLE
Add new development container mit-scheme

### PR DESCRIPTION
**Pull request by Aisuko**
Signed-off-by: aisuko <urakiny@gmail.com>
fixes: #768

### CODE REVIEW

File: containers/mit-scheme/.devcontainer/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# [Choice] MIT-Scheme version: 11.1
+ARG VARIANT=11.1
+FROM mcr.microsoft.com/vscode/devcontainers/mit-scheme:${VARIANT}
```
Comment (2percentsilk) [inline containers/mit-scheme/.devcontainer/Dockerfile:3]: We don't pre-build images from community contributions so this won't work/is unnecessary.

File: containers/mit-scheme/.devcontainer/library-scripts/mit-scheme-debian.sh [hunk 2]
```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
```
Comment (2percentsilk) [inline containers/mit-scheme/.devcontainer/library-scripts/mit-scheme-debian.sh:1]: We recommend moving this file up one level into `containers/mit-scheme/.devcontainer/`.

File: containers/mit-scheme/README.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+## Using this definition with an existing folder
+
+While the definition itself works unmodified, you can select the version of Debian the container uses by updating the `VARIANT` arg in the included `devcontainer.json` (and rebuilding if you've already created the container).
+
+```json
+"args": { "VARIANT": "11.1" }
+```
+
+You can also directly reference pre-built versions of `.devcontainer/base.Dockerfile` by using the `image` property in `.devcontainer/devcontainer.json` or updating the `FROM` statement in your own  `Dockerfile` to one of the following. An example `Dockerfile` is included in this repository.
```
Comment (2percentsilk) [inline containers/mit-scheme/README.md:27]: You'll need to update this to remove pre-build instructions as this image won't be prebuilt by our team.

File: containers/mit-scheme/.devcontainer/Dockerfile (hunk 1 above)
Comment (Aisuko) [reply containers/mit-scheme/.devcontainer/Dockerfile]: Removed the Dockerfile

File: containers/mit-scheme/.devcontainer/library-scripts/mit-scheme-debian.sh (hunk 2 above)
Comment (Aisuko) [reply containers/mit-scheme/.devcontainer/library-scripts/mit-scheme-debian.sh]: Done.

File: containers/mit-scheme/README.md (hunk 3 above)
Comment (Aisuko) [reply containers/mit-scheme/README.md]: Done.

File: containers/mit-scheme/.devcontainer/devcontainer.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+	// "forwardPorts": [],
+
+	// Uncomment to use the Docker CLI from inside the container. See https://aka.ms/vscode-remote/samples/docker-from-docker.
+	// "mounts": [ "source=/var/run/docker.sock,target=/var/run/docker.sock,type=bind" ],
+
+	// Uncomment when using a ptrace-based debugger like C++, Go, and Rust
+	// "runArgs": [ "--cap-add=SYS_PTRACE", "--security-opt", "seccomp=unconfined" ],
+
+	// Comment out connect as root instead. More info: https://aka.ms/vscode-remote/containers/non-root.
+	// "remoteUser": "vscode"
```
Comment (2percentsilk) [inline containers/mit-scheme/.devcontainer/devcontainer.json:27]: We want to make sure this runs as non-root so uncommenting this line would be great!

```suggestion
	"remoteUser": "vscode"
```

Comment (Aisuko) [reply containers/mit-scheme/.devcontainer/devcontainer.json]: Thanks for your quick reply, done.

**Comment by Aisuko** [comment]
Hi @2percentsilk, sorry to bother you, could we merge this PR?

File: containers/mit-scheme/README.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
+# Debian
```
Comment (2percentsilk) [inline containers/mit-scheme/README.md:1]:
```suggestion
# MIT-Scheme (Community)
```

File: containers/mit-scheme/README.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+## Summary
+
+*Simple mit-scheme container with Git installed.*
+
+| Metadata | Value |  
+|----------|-------|
+| *Contributors* | [Aisuko](https://github.com/Aisuko) |
```
Comment (2percentsilk) [inline containers/mit-scheme/README.md:9]:
```suggestion
| *Contributors* | [Aisuko](https://github.com/Aisuko) |
| *Categories* | Community |
```

**Review by 2percentsilk — CHANGES_REQUESTED** [review]
Apologies - two more small changes and then I can merge it!

**Comment by Aisuko** [comment]
Thanks, done.

File: containers/mit-scheme/.devcontainer/devcontainer.json [hunk 7]
```diff
@@ -0,0 +1,28 @@
+{
+	"name": "MIT-Scheme",
```
Comment (2percentsilk) [inline containers/mit-scheme/.devcontainer/devcontainer.json:2]:
```suggestion
	"name": "MIT-Scheme (Community)",
```

**Review by 2percentsilk — APPROVED** [review]

**Pull request merged**